recipes/nyuki_recipe/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "nyuki" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source: 
+  url: https://github.com/00krishna-tools/nyuki/archive/v{{ version }}.tar.gz
+  sha256: 88e943eed59d5ca95d6716b22b47c71a7b7c9323a71348a4623e329bb66d7ae7
+
+build:
+  noarch: python
+  number: 0
+#  skip: true  # [py<36]
+  entry_points:
+    - nyuki = nyuki.nyuki:nyuki
+#  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

I would delete the `build.sh` and `bld.bat` files and use https://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip

```suggestion
  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

---

@chrisburr I actually tried it with the script, and with the 2 files. For some reason the azure pipelines were giving me errors when I just used the script, but no problems with the two files. I did not have a chance to dig into it when I first ran into the issues--was trying to fixed the other issues :). But I can take a look at this again and see if I can get it to run without the 2 extra files.